pom.xml

```diff
@@ -1853,6 +1853,9 @@
                                 <enforceBytecodeVersion>
                                     <maxJdkVersion>${java.version}</maxJdkVersion>
                                     <ignoredScopes>test</ignoredScopes>
+                                    <ignoreClasses>
+                                        <ignoreClass>org.jline.terminal.impl.ffm.*</ignoreClass>
+                                    </ignoreClasses>
```

Comment:
For what reasons, we have to ignore these classes? Any docs, reference, clarification for this?

Reply:
As @pan3793 mentioned: https://github.com/apache/spark/pull/46288

Reply:
OK. But how about bringing the comments as well ? https://github.com/apache/spark/pull/46288/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R2985

Reply:
Sure, I will add the comments.